tests/util/test_unit_conversion.py

```diff
@@ -286,6 +286,18 @@
         (10, UnitOfEnergy.GIGA_JOULE, 10 / 3.6, UnitOfEnergy.MEGA_WATT_HOUR),
         (10, UnitOfEnergy.MEGA_JOULE, 10 / 3.6, UnitOfEnergy.KILO_WATT_HOUR),
         (10, UnitOfEnergy.MEGA_JOULE, 0.010 / 3.6, UnitOfEnergy.MEGA_WATT_HOUR),
+        (10, UnitOfEnergy.KILO_JOULE, 10 / 3.6, UnitOfEnergy.WATT_HOUR),
+        (10, UnitOfEnergy.KILO_JOULE, 0.010 / 3600, UnitOfEnergy.MEGA_WATT_HOUR),
+        (10, UnitOfEnergy.JOULE, 10 / 3600, UnitOfEnergy.WATT_HOUR),
+        (10, UnitOfEnergy.JOULE, 10 / 4.184, UnitOfEnergy.CALORIE),
+        (10, UnitOfEnergy.CALORIE, 0.01, UnitOfEnergy.KILO_CALORIE),
+        (10, UnitOfEnergy.CALORIE, 41.84 / 3600, UnitOfEnergy.WATT_HOUR),
+        (10, UnitOfEnergy.KILO_CALORIE, 0.01, UnitOfEnergy.MEGA_CALORIE),
+        (10, UnitOfEnergy.KILO_CALORIE, 41.84 / 3600, UnitOfEnergy.KILO_WATT_HOUR),
+        (10, UnitOfEnergy.MEGA_CALORIE, 0.01, UnitOfEnergy.GIGA_CALORIE),
+        (10, UnitOfEnergy.MEGA_CALORIE, 41.84 / 3600, UnitOfEnergy.MEGA_WATT_HOUR),
+        (10, UnitOfEnergy.GIGA_CALORIE, 10000, UnitOfEnergy.MEGA_CALORIE),
+        (10, UnitOfEnergy.GIGA_CALORIE, 41.84 / 3.6, UnitOfEnergy.MEGA_WATT_HOUR),
```

Comment:
We should not use any calculation on the expected value as this could hide a conversation error.
I use an online converter to get the expected value, and then the tests ensure that our converter works the same way.